### ISSUE TITLE
Grouped stream entries for content types as Images/Files

### ISSUE THREAD

**Issue by buddh4** [humhub/humhub#3430]
Instead of the Suppression Stream Logic some content types as files or the gallery module should use grouped stream entries containing the latest content.

**E.g. Gallery:**

When uploading multiple images to the gallery module we check if there we've already created an grouped gallery content record within the last day.
If there was no grouped gallery content created by this user within the last day, we create a new one which will render a stream entry with all uploaded images until the next grouped content entry.
If there already is a grouped content entry which is not too old, we don't have to create a new one.

**Comment by luke-** [comment]
This would also be useful for activities like surveys/join space.

**Comment by buddh4** [comment]
Good idea! One issue we have in case of files/images, we need to integrate a way of commenting/liking into the grouped content otherwise users can't comment/like single files/images in the stream.
